### PR TITLE
Update list of BUIPs from github.com/BitcoinUnlimited/BUIP

### PR DESCRIPTION
**Pull request by soeren-b-c**
Updated the BUIP list from the list on github.com/BitcoinUnlimited/BUIP.

Still a few things to clean up, and a few errors to the BitcoinUnlimited/BUIP show them selves.

Feedback welcome of course.

### CODE REVIEW

File: src/data/buips.json [hunk 1]
```diff
@@ -228,7 +236,7 @@
         "title": "Website updates",
         "proposer": "Jerry Chan",
         "submitted": "2016-09-29",
-        "status": "closed",
+        "status": "passed",
```
Comment (sickpig) [inline src/data/buips.json]: according to https://bitco.in/forum/threads/buip-index.1414/ status of `BUIP030` is `closed`

File: src/data/buips.json [hunk 2]
```diff
@@ -565,23 +573,23 @@
         "proposer": "@Norway",
         "submitted": "2017-11-13",
         "status": "closed",
-        "href": "https://github.com/BitcoinUnlimited/BUIP/blob/master/073.mediawiki"
+        "href": "https://github.com/BitcoinUnlimited/BUIP/blob/master/073"
```
Comment (sickpig) [inline src/data/buips.json]: why did you remove `.mediawiki` from the URL? 
(same applies to the following 2 URLs)

File: src/data/buips.json [hunk 3]
```diff
@@ -651,7 +659,7 @@
         "number": "BUIP084",
         "title": "Secure BCH coin splitting instructions",
         "proposer": "@79b79aa8",
-        "submitted": "2018-01-11",
+        "submitted": "2017-01-11",
```
Comment (sickpig) [inline src/data/buips.json]: according to https://bitco.in/forum/threads/buip084-passed-secure-bch-coin-splitting-instructions.8606/ submission data for `BUIP084` is `Jan. 11, 2018`

File: src/data/buips.json [hunk 4]
```diff
@@ -660,119 +668,151 @@
...
     {
-        "number":   "BUIP092",
-        "title":    "Move the official BU repository and issue tracker to GitLab",
+        "number": "BUIP092",
+        "title": "Move the official BU repository and issue tracker to GitLab",
         "proposer": "Gal Buki (@torusJKL)",
-        "submitted":"2018-08-01",
-        "status":   "draft",
-        "href":     "https://github.com/BitcoinUnlimited/BUIP/blob/master/092.mediawiki"
+        "submitted": "2018-08-01",
```
Comment (sickpig) [inline src/data/buips.json]: seems like title of 091 and 092 are mixed, the former is supposed to be about moving to gitlab whereas the latter is about instant txs workshop

File: src/data/buips.json [hunk 5]
```diff
@@ -660,119 +668,151 @@
...
-        "submitted":"2018-08-01",
-        "status":   "draft",
-        "href":     "https://github.com/BitcoinUnlimited/BUIP/blob/master/069.mediawiki"
+        "submitted": "2018-08-01",
+        "status": "closed",
+        "href": "https://github.com/BitcoinUnlimited/BUIP/blob/master/069.mediawiki"
+    },
+    {
+        "number": "BUIP093",
+        "title": "Specification - Graphene Relay",
```
Comment (sickpig) [inline src/data/buips.json]: Initially BUIP 093 has been assigned for graphene v1 spec, later on we decided to move it to actual BU repo (https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/doc/graphene-specification-v1.mediawiki) and reserved it to "Graphene v.2 Improvements and Extensions" (https://bitco.in/forum/threads/buip093-graphene-v-2-improvements-and-extensions.22662/)

**Comment by soeren-b-c** [comment]
@sickpig I have an ugly script that fetches the data from
github.com/BitcoinUnlimited/BUIP
so I will go through your comments and double-check if the errors are in that list as well. Then can update this list for the webpage afterwards.

**Comment by soeren-b-c** [comment]
@sickpig I think this is ready now. Give a shout if you see any more errors / issues.

**Comment by sickpig** [comment]
@soeren-b-c 
I gave another review pass to the BUIP repo and I corrected a bunch of minor nits (mainly updated BUIPs status). 

I opened a PR here: https://github.com/BitcoinUnlimited/BUIP/pull/92

What about re-running your script to update this PR once we got BitcoinUnlimited/BUIP/pull/92 merged?

**Comment by soeren-b-c** [comment]
@sickpig Will do!

**Comment by sickpig** [comment]
> @sickpig Will do!

BitcoinUnlimited/BUIP#92 has been merged.
